compiler/src/iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -138,6 +140,40 @@ getTileAndFuseOptionsFromConfig(func::FuncOp funcOp, int64_t tilingLevel) {
   return options;
 }
 
+/// Default method to initialize the split reduction size and unroll factor in
+/// IREE. These could be overriden by the command line options if specified.
+static bool getSplitReductionOptionsFromConfig(func::FuncOp funcOp,
```

Comment:
Its easier to debug if you return a `FailureOr<...>` or `Optional<..>` . Return by reference is not the prefered coding style.

Reply:
There are two return values.

Reply:
I created a pair. That seems readable enough